canonical/ubuntu.com
# WD-14930 Dev create /partner/intel

## Done

- Dev created /partner/intel
- Removed redirect

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Demo](https://ubuntu-com-14568.demos.haus/partner/intel)
- [Copydoc](https://docs.google.com/document/d/1GstP9FfZfbXO81YcX-9oCJdP9pamlgpl2kKkSqlrWFE/edit?tab=t.0)
- [Design](https://www.figma.com/design/S6KvNUt0agl0bcbp6wjhnB/24.10-Ubuntu.com%2Fpartners?node-id=2225-552&node-type=canvas&m=dev)
## Issue / Card

Fixes # [WD-14930](https://warthogs.atlassian.net/browse/WD-14930)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-14930]: https://warthogs.atlassian.net/browse/WD-14930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
